wodles/aws/aws_s3.py

```diff
@@ -1587,19 +1590,42 @@ def get_ec2_client(self, access_key, secret_key, region, profile_name=None):
 
         boto_session = boto3.Session(**conn_args)
 
+        available_regions = [
+            *boto_session.get_available_regions(
+                service_name=EC2_SERVICE, partition_name='aws',allow_non_regional=True
+            ),
+            *boto_session.get_available_regions(
+                service_name=EC2_SERVICE, partition_name='aws-us-gov',allow_non_regional=True
+            ),
+            *boto_session.get_available_regions(
+                service_name=EC2_SERVICE, partition_name='aws-cn',allow_non_regional=True
+            )
+        ]
```

Comment:
Consider using a constant like `ALL_REGIONS` at the top of the script since this would avoid the API calls and it is the current approach when no region is specified:
https://github.com/wazuh/wazuh/blob/1641c6e903d6b8ccbb1fcbd252101550c16b67fb/wodles/aws/aws_s3.py#L3428-L3430